src/App.js

```diff
@@ -1,5 +1,19 @@
+import styled from 'styled-components/macro'
+import locations from './data.json'
+import LocationsPage from './pages/LocationsPage'
+
 function App() {
-  return <div>Rob is der Geilste!</div>;
+  return (
+    <AppWrapper>
+      <LocationsPage locations={locations} />
+    </AppWrapper>
+  )
 }
 
-export default App;
+export default App
+
+const AppWrapper = styled.section`
```

Comment:
I don't think a `<section>` is the right element, I think a `<div>` would be better here because it is just for styling purpose. I'm not quite sure though.

Reply:
I agree.